src/js/mobile-menu.js

```diff
@@ -0,0 +1,69 @@
+/* global $ */
+(function () {
+
+  let menuSelector;
+  let menuTriggerSelector;
+  let $menuTrigger;
+  let menuTriggerClass;
+  let $menu;
+  let menuSlideoutClass;
+  let menuCloseSelector;
+  let $bodyOverlay;
+  let bodyOverlayClass;
+  let bodyNavOpenClass;
+
+
+  function closeMenu() {
+    $menu.removeClass(menuSlideoutClass);
+    $menuTrigger.removeClass(menuTriggerClass);
+    $bodyOverlay.removeClass(bodyOverlayClass);
+    $('body').removeClass(bodyNavOpenClass);
+  }
+
+  function toggleMenu() {
+    $menu.toggleClass(menuSlideoutClass);
+    $menuTrigger.toggleClass(menuTriggerClass);
+    $bodyOverlay.toggleClass(bodyOverlayClass);
+    $('body').toggleClass(bodyNavOpenClass);
+  }
+
+  function closeMobileMenuHandler() {
+    $(document).click((e) => {
+      const $target = $(e.target);
+
+      // Hide the mobile menu when anything else is clicked
+      if (!$target.closest(menuSelector).length && !$target.closest(menuTriggerSelector).length) {
+        if ($menu.width() > 200) {
+          closeMenu();
+        }
+      }
+    });
+  }
+
+  function closeMobileMenuIconHandler() {
+    menuCloseSelector.click(closeMenu);
+  }
+
+  function setupEventHandlers() {
+    $(menuTriggerSelector).click(toggleMenu);
+  }
+
+  function init() {
+    menuSelector = '.today-site-nav';
+    menuTriggerSelector = '.navbar-toggler';
+    $menuTrigger = $(menuTriggerSelector);
+    menuTriggerClass = 'active';
+    $menu = $(menuSelector);
+    menuSlideoutClass = 'slideout';
+    menuCloseSelector = $menu.find('.close');
```

Comment:
Since this variable (`menuCloseSelector`) references a jQuery object, it should be named with a `$` prefix to help distinguish it as such.

Might make more sense to give it a name that helps clarify it is a jQuery object as well, vs a selector string; maybe `$menuCloseBtn`?

Reply:
Fixed in 6cecafb